python/test/t_DistributionFactory_std.py

```diff
@@ -46,9 +47,56 @@
             dist = factory.build(sample)
         except Exception:
             ok = True
-        print(weird + ":", "ok" if ok else "fail")
-        assert ok, str(factory) + " vs " + weird
+            print(weird + ":", "ok" if ok else "fail")
+            assert ok, str(factory) + " vs " + weird
 
 factory = ot.DistributionFactory.GetByName("PoissonFactory")
 print(factory)
 assert factory.getImplementation().__class__.__name__ == "PoissonFactory", "wrong name"
+
+# Now, test each factory for the partial estimation of any subset of its parameters
+
+params = {
+    "BetaFactory": 0.05,
+    "FisherSnedecorFactory": 0.25,
+    "LogisticFactory": 0.025,
+    "LogNormalFactory": 0.02,
+    "ParetoFactory": 0.15,
+    "RiceFactory": 0.3,
+    "StudentFactory": 0.15,
+    "StudentFactory": 0.15,
+    "TrapezoidalFactory": 1.0,
+    "TriangularFactory": 0.025,
+    "VonMisesFactory": 0.02,
+    "NegativeBinomialFactory": 0.05,
+}
+
+for factory in factories:
+    print("factory=", factory)
+    try:
+        factoryName = factory.getImplementation().getClassName()
+        if factoryName in params:
+            eps = params[factoryName]
```

Comment:
you could use `dict.get(key, default_value)` which is much shorter than the if/else

Reply:
done, thks